src/language-js/printer-estree.js

```diff
@@ -5121,7 +5121,7 @@ function isObjectType(n) {
 
 // eg; `describe("some string", (done) => {})`
 function isTestCall(n) {
-  const unitTestRe = /^(f|x)?(it|describe|test)$/;
+  const unitTestRe = /^(f|x)?(it|describe|test|skip)$/;
```

Comment:
Thanks for making a PR! This will also match `fskip` and `xskip`; could you adjust the regexp so it won't? I think `/^(skip|(f|x)?(it|describe|test))$/` should do it.